### PR TITLE
Bump version to 2.4.0.

### PR DESCRIPTION
**Pull request by troessner**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
+# 2.4.0 / 2015-12-26
+
+* [FEATURE] Add progress bar functionality (by Nate Holland)
+
 # 2.3.0 / 2015-11-30
 
-* [FEATURE] Added global score calculation
+* [FEATURE] Added global score calculation (by Ancor Gonzalez Sosa)
```
Comment (troessner) [inline CHANGELOG.md]: Seems like we forgot to give credit on our last release ;)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
+# 2.4.0 / 2015-12-26
+
+* [FEATURE] Add progress bar functionality (by Nate Holland)
```
Comment (troessner) [inline CHANGELOG.md]: Thanks @natesholland !

File: CHANGELOG.md (hunk 1 above)
Comment (Onumis) [inline CHANGELOG.md]: oh yeah! of course :+1:

**Comment by Onumis** [comment]
Published!

**Comment by troessner** [comment]
Awesome!

**Pull request merged**